src/lib.rs

```diff
@@ -194,7 +194,16 @@ pub async fn collect_from_nodes(
             context: "list nodes".to_string(),
             source,
         })?;
-    for node in nodes.items {
+    compute_node_utilizations(nodes, resources).await?;
+    Ok(())
+}
+
+#[instrument(skip(node_list, resources))]
+pub async fn compute_node_utilizations(
+    node_list: ObjectList<Node>,
+    resources: &mut Vec<Resource>,
+) -> Result<(), Error> {
```

Comment:
wrong name (maybe caused by a wrong copy/paste of the error at line 217, that should be `ResourceQualifier::Allocatable` and not `ResourceQualifier::Utilization`, can you change it in this PR (else I'll update master))
A better name could be `search_allocatable_from_nodes` (nothing related with utilization) (search or extract,... but not compute because too vague and no real mathematics computation)

Reply:
Sure will fix. wasn't really sure about the naming 😂

Reply:
I replace the wrong `ResourceQualifier::Utilization` by `ResourceQualifier::Allocatable` on master